commit fc308197f78944b056254390932ce891bd5ea36f
Author: root <root@dhcp-flab2-52.cesnet.cz>
Date:   Wed Jun 21 12:57:17 2017 +0200

    fix query semantic ?
    
     (DEBUG) eb6f060f/getEvents: execute: ('SELECT event_id as id FROM last_events WHERE client_id = %s ORDER BY last_events.id DESC LIMIT 1', 1L) {}
     (INFO) eb6f060f/getEvents: cannot getLastReceivedId - TypeError: argument 2 to map() must support iteration

diff --git a/warden3/files/opt/warden_server/warden_server.py b/warden3/files/opt/warden_server/warden_server.py
index 0c1e55c..d4a694b 100755
--- a/warden3/files/opt/warden_server/warden_server.py
+++ b/warden3/files/opt/warden_server/warden_server.py
@@ -757,7 +757,7 @@ class MySQL(ObjectBase):
 
 
     def getLastReceivedId(self, client):
-        res = self.query("SELECT event_id as id FROM last_events WHERE client_id = %s ORDER BY last_events.id DESC LIMIT 1", client.id, commit=True).fetchall()
+        res = self.query("SELECT event_id as id FROM last_events WHERE client_id = %s ORDER BY last_events.id DESC LIMIT 1", [client.id], commit=True).fetchall()
         try:
             row = res[0]
         except IndexError:
